README.md

```diff
@@ -45,7 +45,6 @@ To use Gradle, add the following lines to your build.gradle file:
 ```gradle
 repositories {
     mavenCentral()
-    google()
```

Comment:
we probably want to leave this here until we're ready to release 1.30.9

Reply:
Ah yeah - reverted!